app/templates/app/image.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
```

Comment:
Dies hab ich nur hinzugefügt weil es generell keine Seite zum Anzeigen von Bildern gibt. Generell gestalten viele Webhoster ihre Seiten so, dass sie die Bilder in ihreme Layout der Webseite anzeigen lassen können. Siehe z.B. devianart.com (http://selenada.deviantart.com/art/Water-Splash-Adoptable-Portrait-573306409)